sdk/openai/openai/samples/cookbook/speechToSpeechChat/webpack.config.js

```diff
@@ -0,0 +1,55 @@
+// Generated using webpack-cli https://github.com/webpack/webpack-cli
+
+const path = require('path');
```

Comment:
Consistency desired here with 2 spaces for JS files here such as with those in the src folder